src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/media-edit-overlay/main.js

```diff
@@ -200,8 +202,10 @@ define([
                 translate: this.sandbox.translate,
                 formatBytes: this.sandbox.util.formatBytes,
                 crop: this.sandbox.util.cropMiddle,
-                icon: fileIcons.getByMimeType(media.mimeType)
+                icon: fileIcons.getByMimeType(media.mimeType),
+                thumbnailFormat: constants.thumbnailFormat
             }));
+            this.removePlaceholderOnImgLoad($info, fileIcons.getByMimeType(media.mimeType));
```

Comment:
I would save the result of the `fileIcons.getByMimeType` in a variable, so that the call doesn't have to be done twice.